src/vm/peimage.inl

```diff
@@ -481,8 +481,19 @@ inline BOOL PEImage::HasNativeHeader()
         return GetLoadedLayout()->HasNativeHeader();
     else
     {
-        PEImageLayoutHolder pLayout(GetLayout(PEImageLayout::LAYOUT_ANY,LAYOUT_CREATEIFNEEDED));
```

Comment:
@gkhanna79 I do not have an arm32 platform to verify this locally (although the arm32 CI passed with this change.). I will amend the change to be ARM64 specific.  arm32 contribs can reference this if they need similar fixes.